catalog/core/catalog-core-commands/src/test/java/org/codice/ddf/commands/catalog/RemoveCommandTest.java

```diff
@@ -128,4 +171,21 @@ protected SolrCacheMBean getCacheProxy() {
 
         return metacards;
     }
+
+    private java.util.List<Result> getResultList(int amount) {
```

Comment:
Comment or rename to indicate operation of adding random UUID to each result in the list.

Perhaps also changing "amount" to "size" would be a bit more consistent with how we usually write these kinds of list population methods. 

Reference: https://github.com/codice/ddf/pull/2148/files#diff-84f7749e6d7b0fc4c56387a17d2799a8R176